pkg/command/service/load.go

```diff
@@ -0,0 +1,423 @@
+// Copyright 2021 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package service
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+
+	"knative.dev/kperf/pkg/command/utils"
+
+	"log"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
+
+	v1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+
+	"k8s.io/apimachinery/pkg/watch"
+
+	"knative.dev/kperf/pkg"
+	"knative.dev/serving/pkg/apis/serving"
+	servingv1 "knative.dev/serving/pkg/apis/serving/v1"
+	servingv1client "knative.dev/serving/pkg/client/clientset/versioned/typed/serving/v1"
+)
+
+const (
+	LoadOutputFilename = "ksvc_loading_time"
+)
+
+func NewServiceLoadCommand(p *pkg.PerfParams) *cobra.Command {
+	loadArgs := pkg.LoadArgs{}
+	serviceLoadCommand := &cobra.Command{
+		Use:   "load",
+		Short: "Load and Measure Knative service",
```

Comment:
I will suggest to use `Load test and Measure Knative service` because `Load` could be a little confused.

Reply:
Thank you, it's a good idea.